src/scene/end.js

```diff
@@ -0,0 +1,38 @@
+import kbm from '../config/kaboom.js';
+import Sound from '../utils/Sound.js';
+
+function showEndingPage(pageType) {
+  const canvas = document.querySelector('canvas');
+  canvas.style.display = 'none';
+  const endingSection = document.querySelector('.ending');
+  endingSection.style.display = 'block';
+
+  Sound.pauseMusic('backgroundMusic');
+
+  if (pageType === 'win') {
+    Sound.playMusic('winMusic');
+    document.querySelector('.win').style.display = 'block';
+  } else {
+    Sound.playMusic('loseMusic');
+    document.querySelector('.lose').style.display = 'block';
+  }
+
+  kbm.quit();
+}
+
+function socialMediaHandler(spaceshipHealth, completionPercentage) {
+  const twitter = document.querySelector('#twitter');
+  let text =
+    'Earth faces an alien invasion, putting HackerRank in jeopardy. Help Hari in taking all HackerRankers to safety in HackerSpace. @sp2hari @hackerrank\n\n';
```

Comment:
**question:** Lol. @sp2hari Are you okay with this `@sp2hari` mention on twitter? 🙂